### ISSUE TITLE
Incomplete yt initial data received

### ISSUE THREAD

**Issue by NinTuta** [axcore/tartube#585]
(Translated with Google)
What am I doing wrong? under "Output" I get the following:

/home/ma/.local/bin/yt-dlp --newline -i --match-filter !is_live -r 850K --hls-prefer-native --restrict-filenames --write-description --cookies "/media/ma/HDD 02/10 TarTube Videos/cookies.txt" --write-thumbnail --write-sub --windows-filenames --no-overwrites --sub-lang ar,en,de --output "/media/ma/HDD 02/10 TarTube Videos/The Educated Barfly/%(title)s.%(ext)s" --verbose https://www.youtube.com/@TheEducatedBarfly/videos
[debug] Command-line config: ['--newline', '-i', '--match-filter', '!is_live', '-r', '850K', '--hls-prefer-native', '--restrict-filenames', '--write-description', '--cookies', '/media/ma/HDD 02/10 TarTube Videos/cookies.txt', '--write-thumbnail', '--write-sub', '--windows-filenames', '--no-overwrites', '--sub-lang', 'ar,en,de', '--output', '/media/ma/HDD 02/10 TarTube Videos/The Educated Barfly/%(title)s.%(ext)s', '--verbose', 'https://www.youtube.com/@TheEducatedBarfly/videos']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8 (No ANSI), error utf-8 (No ANSI), screen utf-8 (No ANSI)
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-6.2.0-26-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.46.0, pyxattr-0.7.2, secretstorage-3.3.1, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: ()
[debug] Loaded 1855 extractors
[youtube:tab] Extracting URL: https://www.youtube.com/@TheEducatedBarfly/videos
[youtube:tab] @TheEducatedBarfly/videos: Downloading webpage
WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (1/3)...
[youtube:tab] @TheEducatedBarfly/videos: Downloading webpage
WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (2/3)...
[youtube:tab] @TheEducatedBarfly/videos: Downloading webpage
WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (3/3)...
[youtube:tab] @TheEducatedBarfly/videos: Downloading webpage
WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube:tab] Incomplete yt initial data received; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U. Giving up after 3 retries
[debug] [youtube:tab] Extracted SAPISID cookie
ERROR: [youtube:tab] @TheEducatedBarfly: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check

**Comment by NinTuta** [comment]
I was able to fix the problem in the meantime.

**Comment by axcore** [comment]
I tested [your URL](https://www.youtube.com/@TheEducatedBarfly/videos) with yt-dlp alone, the videos downloaded without problems.

You don't say how you fixed the problem, but I can't see any problems with your download options, perhaps it was something in the cookies file?

**Comment by NinTuta** [comment]
Sorry I didn't say how I fixed the problem.
First I installed Tartube on a second identical system. It worked flawlessly on the second system.
Then I compared the two settings under "Edit" and adopted them from the new system.
I noticed that the "Firefox" browser under "Cookies" could not be removed without replacement.
In the first attempt I replaced "Firefox" with "Chomium" and then removed "Chromium".
I suspect the problem is related to "Firefox" or my "Firefox" settings or their "Add-ons"
